specs/engage-digital_postman2.json

```diff
@@ -7842,8 +7848,8 @@
                     "value": "Bearer {{ENGAGE_DIGITAL_ACCESS_TOKEN}}"
                   },
                   {
-                    "key": "Accept",
-                    "value": "application/json"
+                    "key": "Content-Type",
+                    "value": "application/x-www-form-urlencoded"
```

Comment:
This was generated automatically and seems to be correct as `"key": "Content-Type"` is duplicated.